content/client-server-api/modules/mentions.md

```diff
@@ -45,14 +69,19 @@ Clients should display mentions differently from other elements. For
 example, this may be done by changing the background color of the
 mention to indicate that it is different from a normal link.
 
-If the current user is mentioned in a message (either by a mention as
-defined in this module or by a push rule), the client should show that
+If the current user is mentioned in a message, the client should show that
 mention differently from other mentions, such as by using a red
 background color to signify to the user that they were mentioned.
 
 When clicked, the mention should navigate the user to the appropriate
 user or room information.
 
+TODO Behavior with replies.
+
+TODO Behavior with edits.
```

Comment:
Does it make sense to include information here about handling replies / edits or in their respective sections? I think here (and then cross-reference), but I'm not sure.

Reply:
My instinct would actually be to go the other way around (keep the mentions section simple, and put the thorny details of the interactions with replies and edits in their respective sections). There is already an [edits of replies](https://spec.matrix.org/v1.6/client-server-api/#edits-of-replies) section, and by analogy I think an "edits of mentions" section would make sense.

Not a strong opinion though.

Reply:
Edits of replies which include mentions does get thorny! 👍 That sounds like a better place indeed! Thank you!